### ISSUE TITLE
Maui on Android.

### ISSUE THREAD

**Issue by ingnelson** [rousan/maui#1]
Hello.

Would be great if you create a android app where we can use Maui to create a tunnel like a VPN on Android .

**Comment by rousan** [comment]
maui is meant for file transfer actually. Did you mean transferring data or files between devices through maui like from Desktop/PC to Android device?

**Comment by ingnelson** [comment]
@rousan yes , I thought Maui could transfer data .

**Comment by rousan** [comment]
Yes, but it does through TCP tunneling, you can check out this app to transer data/files: https://datash.co/

**Comment by ingnelson** [comment]
@rousan what I mean it is I can use a Maui on my PC (client) or Android ( client ) then Install Maui on a VPS (server) to connect to internet like a VPN .
